files/main.sh

```diff
@@ -44,18 +53,4 @@ curl "<%= script.url %>" | /bin/bash
   <% end %>
 <% end %>
 
-mkdir -p /tmp/configs
```

Comment:
The `configs` section was really just an example of how the `files` feature could be used, as the `scripts` section was as well. Do you have existing stuff making use of it at the moment? Happy to add it or something similar back if so.